weather_dl_v2/cli/app/subcommands/queue.py

```diff
@@ -26,6 +26,9 @@ class QueueValidator(Validator):
     pass
 
 
+queue_key_order = ['client_name', 'license_id', 'queue']
```

Comment:
nit: license_id, client_name, queue

Reply:
updated in [7887b24](https://github.com/google/weather-tools/pull/424/commits/7887b243c8daf71792e56390c2f561f307bf24f0)